sim_assiants/FootContact[] foot_contacts
